### PR TITLE
check operating system

### PR DESCRIPTION
**Pull request by proszewskip**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,4 +1,10 @@
 MONO_SDK=$(pwd)/mono-wasm-sdk
 
-dotnet build Inzynierka.sln
+platform="$(uname -s)"
+case "${platform}" in
+    MSYS*)     buildCmd="dotnet build";;
+    *)          buildCmd="xbuild";;
```
Comment (Gelio) [inline build.sh]: I got a message that `xbuild` is deprecated while using it and that I should use `msbuild` :smile: Could you verify if you get the same message and if so, change the script?

**Review by Gelio — CHANGES_REQUESTED** [review]
By the way, how do you run this script on Windows?

```bash
bash build.sh
```

using Cmder?

File: build.sh (hunk 1 above)
Comment (proszewskip) [reply build.sh]: I do not have `msbuild` on my linux machine. There is also no message that `xbuild` is deprecated.

**Comment by proszewskip** [comment]
Yes, I run it with `bash build.sh`.

File: build.sh (hunk 1 above)
Comment (proszewskip) [reply build.sh]: I have it after updating mono 👍 .

**Review by Gelio — APPROVED** [review]
Good job :smile: Janek should approve such use of bash scripts :dancing_men:

**Pull request merged**